blocks/search/index.js

```diff
@@ -7,16 +7,17 @@
 /**
  * Internal block libraries.
  */
+const { registerBlockType } = wp.blocks;
+const { RichText } = wp.editor;
 const { __ } = wp.i18n;
-const { registerBlockType, RichText } = wp.blocks;
 
 /**
  * Register block.
  */
 export default registerBlockType(
 	'amp-travel/search',
 	{
-		title: __( 'Search' ),
+		title: __( 'Travel Search' ),
```

Comment:
There's a 'Search' block in Core, so maybe this will help to distinguish it.

Reply:
Ideally we could also create a separate category for Travel blocks, but I think it's not that critical for the demo purposes at this moment.